docs/get-started/install.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Install
+parent: Get Started
+nav_order: 1
+---
+
+# Install ONNX Runtime
+
+Use this guide to install ONNX Runtime and its dependencies, for your target operating system, hardware, accelerator, and language.
+
+## Prerequisites
+
+### Linux / CPU
+
+1. English language package with the `en_US.UTF-8` locale
+
+    * Install [language-pack-en package](https://packages.ubuntu.com/search?keywords=language-pack-en)
+    * Run `locale-gen en_US.UTF-8`
+    * Run `update-locale LANG=en_US.UTF-8`
+
+2. OpenMP
+
+   * `apt-get install libgomp1`, which installs **libgomp.so.1**
+
+### Linux / GPU
+
+1. English language package with the `en_US.UTF-8` locale
+
+    * Install [language-pack-en package](https://packages.ubuntu.com/search?keywords=language-pack-en)
+    * Run `locale-gen en_US.UTF-8`
+    * Run `update-locale LANG=en_US.UTF-8`
+
+2. CUDA 10.1 and cuDNN 7.6.5
+
+    * Version dependencies from older ONNX Runtime releases can be found in [prior release notes](https://github.com/microsoft/onnxruntime/releases).
+
+### Windows / CPU
+
+1. English language package with the `en_US.UTF-8` locale
+
+2. [Visual C++ 2019 runtime](https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads)
+
+3. OpenMP
+    * Comes installed with Visual C++ runtime
+    * Or is also available as redist packages [vc_redist.x64.exe](https://aka.ms/vs/16/release/vc_redist.x64.exe) and [vc_redist.x86.exe](https://aka.ms/vs/16/release/vc_redist.x86.exe)
+  
+### Windows / GPU
+
+1. English language package with the `en_US.UTF-8` locale
+
+2. [Visual C++ 2019 runtime](https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads)
+
+3. CUDA 10.1 and cuDNN 7.6.5
+    * Version dependencies from older ONNX Runtime releases can be found in [prior release notes](https://github.com/microsoft/onnxruntime/releases).
+
+## Install
+
+* Default CPU Provider (Eigen + MLAS)
+* GPU Provider - NVIDIA CUDA
+* GPU Provider - DirectML (Windows)
+  * *On Windows, the [DirectML execution provider](https://github.com/microsoft/onnxruntime/tree/master/docs/execution_providers/DirectML-ExecutionProvider.md) is recommended for optimal performance and compatibility with a broad set of GPUs.*
+
+If using pip, run `pip install --upgrade pip` prior to downloading.
+
+|Repository|Official build|Nightly build|
```

Comment:
I think the table/matrix on the onnxruntime.ai main page is pretty useful for figuring out installation instructions; may be useful to include a link to that page.

Reply:
Yes, good idea. Ultimately we might want to combine these.